dev/archery/archery/crossbow/reports.py

```diff
@@ -83,6 +84,29 @@ def tasks(self):
     def show(self):
         raise NotImplementedError()
 
+    @property
+    def rows(self):
+        """
+        Produces a generator that allow us to iterate over
+        the job tasks as a list of rows.
+        """
+        for task_name, task in sorted(self.job.tasks.items()):
+            task_status = task.status()
+            row = [
+                task_name,
+                task_status.combined_state,
+                task_status.build_links,
+                self.branch_url(task.branch),
+                task.ci,
+                # We want this to be serialized as a dict instead
+                # of an orderedict.
+                {k: v for k, v in task.params.items()},
+                task.template,
+                # Arrow repository commit
+                self.job.target.head
```

Comment:
I asked about a commit, but I see here that it is already included! It seems that I missed that there was more content in https://gist.github.com/raulcd/1799c8499ef228dfdf1ab0d986367d2a (only the first three columns are visible ;))

Reply:
yes, the view on the gist for CSV is not great :)